extensions/amp-story/1.0/simple-template.js

```diff
@@ -83,25 +83,25 @@ function renderSingle(doc, elementDef) {
   const hasLocalizedTextContent = hasOwn(elementDef, 'localizedStringId');
   const hasLocalizedLabel = hasOwn(elementDef, 'localizedLabelId');
   if (hasLocalizedTextContent || hasLocalizedLabel) {
-    const win = toWin(doc.defaultView);
-    Services.localizationServiceForOrNull(win).then((localizationService) => {
-      devAssert(localizationService, 'Could not retrieve LocalizationService.');
+    const localizationService = Services.localizationForDoc(
+      devAssert(doc.body)
```

Comment:
Should we fix this one as well?

Reply:
hmm I think this should still work? we're just using the document object. this would still work for shadow DOM